dev/io.openliberty.microprofile71.internal_fat/fat/src/io/openliberty/microprofile71/internal/test/suite/MPCompatibilityTestUtils.java

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
```

Comment:
If this file is mostly unchanged we should retain the original copyright date.

For the others, you could argue that you've changed everything important about the file, but this one appears to contain lots of code that's unchanged.